src/test/java/fr/techad/edc/httpd/utils/TokenUtilsTest.java

```diff
@@ -0,0 +1,45 @@
+package fr.techad.edc.httpd.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TokenUtilsTest {
+	private final String tokenPath = "./token.info";
+	TokenUtils tutil = TokenUtils.getInstance();
+	@Test
+	public void keyCreationLength() {
+		Assert.assertEquals(24, tutil.genSecretKey().length());
+	}
+
+	@Test
+	public void keyCreationBlank() {
+		Assert.assertFalse(tutil.genSecretKey().isBlank());
```

Comment:
This test is similar to the previous because the test check the length is equal to 24 so it not blank. You can remove it